source/slang/slang-lower-to-ir.cpp

```diff
@@ -8555,7 +8546,16 @@ RefPtr<IRModule> generateIRForTranslationUnit(
     // marked for mandatory "early" inlining.
     //
     performMandatoryEarlyInlining(module);
-
+    
```

Comment:
We should run `simplifyIR()` before running the jpv passes to clean up deadcode.

Reply:
We can also move this pass after the `constructSSA` pass.